### PR TITLE
Add script for updating advent of code leaderboard stats

### PR DESCRIPTION
**Pull request by ni5arga**
Added script for updating advent of code private leaderboard stats for AoC 2023 in a post of the subreddit.

### CODE REVIEW

File: aoc/main.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import praw
+import requests
+import os
+
+client_id = os.environ["REDDIT_CLIENT_ID"]
+client_secret = os.environ["REDDIT_CLIENT_SECRET"]
+reddit_pass = os.environ["REDDIT_PASSWORD"]
+username = os.environ["REDDIT_USERNAME"]
+user_agent = 'AdventOfCode Leaderboard Updater (by https://github.com/ni5arga/)'
+aoc_session_cookie = os.environ["AOC_SESSION_COOKIE"]
```
Comment (Bhupesh-V) [inline aoc/main.py:10]: LGTM 👌🏽 

Can you add a small README detailing how to get this cookie value?
[Is this the correct way](https://github.com/wimglenn/advent-of-code-wim/issues/1#issue-193321235), or any other trick?

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]

File: aoc/main.py (hunk 1 above)
Comment (ni5arga) [reply aoc/main.py:10]: Done, added.

**Review by Bhupesh-V — APPROVED** [review]
Nice work 🔥

**Comment by Bhupesh-V** [comment]
@all-contributors please add @ni5arga  for code

**Pull request merged**